plugins/nf-schema/src/main/nextflow/validation/SamplesheetConverter.groovy

```diff
@@ -44,17 +44,23 @@ class SamplesheetConverter {
         this.meta.size() > 0
     }
 
-    private static List unusedHeaders = []
+    private static List unrecognisedHeaders = []
 
-    private static addUnusedHeader (String header) {
-        this.unusedHeaders.add(header)
+    private static addUnrecognisedHeader (String header) {
+        this.unrecognisedHeaders.add(header)
     }
 
-    private static logUnusedHeadersWarning(String fileName) {
-        def Set unusedHeaders = this.unusedHeaders as Set
-        if(unusedHeaders.size() > 0) {
-            def String processedHeaders = unusedHeaders.collect { "\t- ${it}" }.join("\n")
-            log.warn("Found the following unidentified headers in ${fileName}:\n${processedHeaders}" as String)
+    private static logUnrecognisedHeaders(String fileName) {
+        log.info("HEADERS: ${this.unrecognisedHeaders}" as String)
```

Comment:
I think we don't need this, we already have the warning or error messages

Reply:
Oh yeah that was a debug statement that I left in, my bad :grin: